.github/workflows/rust.yml

```diff
@@ -176,6 +175,19 @@ jobs:
     - name: Check for unexpected chain load operations
       run: |
         ./scripts/check_chain_loads.sh
+
+  check-copyright-headers:
```

Comment:
nit: Maybe `lint-check-copyright-headers`. In fact `lint-cargo-clippy` is probably logical after that